### DISCUSSION TITLE
Multiple selection pattern in the 3D panel

### DISCUSSION THREAD

**Discussion by banisadr** [foxglove/community#415] [Q&A] [answered]
Ability to select multiple objects in 3D Panel and inspect their details using inspect objects tool.

**Comment by jtbandes** [comment, marked as answer]
Does the potential solution described in https://github.com/foxglove/studio/issues/4826#issuecomment-1319110319 make this a non-issue? Or do we need both?

**Reply by banisadr** [reply]
The solution landed on in the [customize selected_id PR](https://github.com/foxglove/studio/pull/4944) functionally solves this.
